nomt/src/store/mod.rs

```diff
@@ -24,7 +24,7 @@ use std::{
 use std::os::unix::fs::OpenOptionsExt as _;
 
 pub use self::page_loader::{PageLoad, PageLoader};
-pub use bitbox::BucketIndex;
+pub use bitbox::{BucketIndex, BucketUtilization as HashTableUtilization};
```

Comment:
I'm not a fun of this alias, I think that just having the struct named `HashTableUtilization` would be nice.